test/pom.xml

```diff
@@ -73,10 +82,17 @@ THE SOFTWARE.
     </dependencies>
   </dependencyManagement>
   <dependencies>
+    <!-- For maven-hpi-plugin -->
+    <dependency>
+      <groupId>${project.groupId}</groupId>
+      <artifactId>jenkins-core</artifactId>
+      <version>${project.version}</version>
+      <scope>provided</scope>
+    </dependency>
```

Comment:
Used by the new `TestRuntime` mojo in `maven-hpi-plugin` to detect the Jenkins core version and fill in the values of `jenkins.addOpens` and `jenkins.insaneHook`. This matches `plugin-pom`.